src/views/Header/Header.js

```diff
@@ -281,14 +282,22 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
       screenProps,
       progress,
       style,
+      isLandscape,
       ...rest
     } = this.props;
 
     const { options } = this.props.getScreenDetails(scene);
     const headerStyle = options.headerStyle;
+    const statusBarHeight = isLandscape ? 0 : STATUSBAR_HEIGHT;
```

Comment:
Maybe we want to rename `statusBarHeight` if there is already a variable named `STATUSBAR_HEIGHT`?  Maybe `landscapeAwareStatusBarHeight`? Might be confusing for others looking at the code base seeing two variables both for status bar height.

Reply:
Good point. Fixed!